storm/src/main/storm/mesos/MesosNimbus.java

```diff
@@ -107,13 +111,24 @@
   public static final String CONF_MESOS_CONTAINER_DOCKER_IMAGE = "mesos.container.docker.image";
   public static final String CONF_MESOS_SUPERVISOR_STORM_LOCAL_DIR = "mesos.supervisor.storm.local.dir";
   public static final String FRAMEWORK_ID = "FRAMEWORK_ID";
+  public static final String DEFAULT_MESOS_COMPONENT_NAME_DELIMITER = "|";
+
+  public static final String CONF_ZOOKEEPER_SERVERS = "storm.zookeeper.servers";
+  public static final String CONF_ZOOKEEPER_PORT = "storm.zookeeper.port";
+  public static final String CONF_STORM_LOGVIEWER_ZK_DIR = "storm.logviewer.zookeeper.dir";
```

Comment:
I haven't fully read every line of code yet (my initial review was focusing on touchpoints that I was already familiar with), but I'm assuming that anything needing to create a file in ZK would go through a `mkdir -p` type of operation in ZK to ensure the parent dir already exists.  So I wouldn't expect an explicit and separate "create the root dir in ZK" piece of logic.  I would expect that to happen naturally whenever *anything* goes to store state in ZK for storm-mesos itself.